inc/Plugin.php

```diff
@@ -0,0 +1,256 @@
+<?php
+
+/*
+Gravity Forms Term Select main plugin class
+
+@todo
+- option to allow new terms
+- additional options available thru chosen
+- can we use the chosen version bundled with GF? will our copy conflict?
+*/
+
+namespace OomphInc\GFormsTermSelect;
+
+class Plugin {
+
+	const type = 'term-select'; // gforms field type
```

Comment:
To be fair, they don't mention class constants specifically! The filename is also not `class-plugin.php`